### PR TITLE
fix: Improve and standardize `Script` module `vfuncs`

### LINKED ISSUE #1386: script: Filename and folder name testing is too restrictive and unsecure
Currently, the regular expressions defined in the script.py module (at [this location](https://github.com/viur-framework/viur-core/blob/develop/src/viur/core/modules/script.py#L11)) do not allow spaces in filenames or folder names. This restriction may prevent the use of scripts in directories or files with spaces in their names, which is a common scenario.

**Expected Behavior**
The regular expressions should accommodate spaces in both filenames and folder names. For example:

A filename like my script.py should be valid.
A folder like project files/ should also be valid.
Suggested Changes
Update the regular expressions in the script.py file to include support for spaces. Specifically:

Allow spaces (\s) where filenames or folder names are defined in the regular expression.
Ensure that this change does not break any existing functionality or introduce regressions.
Additional Notes

Be mindful of potential edge cases where spaces at the beginning or end of filenames/folder names could cause issues.
If applicable, add or update unit tests to validate the new behavior.
Environment

Thank you for considering this improvement!

### LINKED ISSUE #1387: script: `vfunc` implemented wrong on `name` bone
The validation function (vfunction) for both filenames and folder names does not return human-readable error messages when invalid inputs are provided. Instead, the error message shown is simply "errors.true". This behavior is observed in the following lines:

[Filename validation](https://github.com/viur-framework/viur-core/blob/69ee3b246645b73d161d796cdab4c66f2bcc2e64/src/viur/core/modules/script.py#L66)
[Folder name validation (a few lines above)](https://github.com/viur-framework/viur-core/blob/69ee3b246645b73d161d796cdab4c66f2bcc2e64/src/viur/core/modules/script.py)

**Current Behavior**
When an invalid filename or folder name is entered, the error message displayed is "errors.true", which is not informative for the user.

**Expected Behavior**
The validation function should return meaningful, human-readable error messages for both filenames and folder names, such as:

For filenames: "Invalid filename. Filenames must not contain invalid characters or spaces."
For folder names: "Invalid folder name. Folder names must not contain invalid characters or spaces."
This will improve the user experience and make it easier to understand and correct errors.

**Additional Notes**
Include specific reasons for validation failure in the error messages (e.g., "Filenames cannot contain spaces").
Review any related documentation or test cases that reference this behavior to ensure consistency.

### PR DESCRIPTION
**Pull request by phorward**
- Fixes #1386
- Fixes #1387

### CODE REVIEW

File: src/viur/core/modules/script.py [hunk 1]
```diff
@@ -63,12 +59,16 @@ class ScriptLeafSkel(BaseScriptAbstractSkel):
...
+        vfunc=lambda value:
+            None if File.is_valid_filename(value) and value.endswith(".py") and value.removesuffix(".py")
+            else "Filename is invalid or doesn't have a '.py'-suffix",
     )
 
     script = RawBone(
         descr="Code",
-        indexed=False
+        indexed=False,
+        required=True,
```
Comment (sveneberth) [inline src/viur/core/modules/script.py:71]: On EVERY filesystem, even on windows, I can create an empty file. Why should this no longer by possible in ViUR.

```suggestion
```

**Review by sveneberth — CHANGES_REQUESTED** [review]

**Review by sveneberth — APPROVED** [review]

**Review by ArneGudermann — APPROVED** [review]
